docs/api-reference/sqliterdb.md

```
@@ -301,15 +301,20 @@ def get(
 **Parameters:**
 
-| Parameter           | Type      | Default    | Description           |
-| ------------------- | --------- | ---------- | --------------------- |
-| `model_class`       | `type[T]` | *required* | The model class       |
-| `primary_key_value` | `int`     | *required* | The primary key value |
+| Parameter           | Type           | Default    | Description                                   |
+| ------------------- | -------------- | ---------- | --------------------------------------------- |
+| `model_class`       | `type[T]`      | *required* | The model class                               |
+| `primary_key_value` | `int`          | *required* | The primary key value                         |
+| `bypass_cache`      | `bool`         | `False`    | Skip cache read/write for this lookup         |
+| `cache_ttl`         | `int` &#124; `None` | `None` | Optional TTL override for this specific lookup |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)